data-prepper-logstash-configuration/src/main/resources/org/opensearch/dataprepper/logstash/mapping/opensearch.mapping.yaml

```diff
@@ -0,0 +1,7 @@
+pluginName: opensearch
```

Comment:
Please add an integration test case for this.

See the `LogstashConfigConverterIT` Javadocs for how to add a new case. It only requires creating input and expected files.